bash_completion

```diff
@@ -17,11 +17,11 @@ __nvm_commands ()
 
   current_word="${COMP_WORDS[COMP_CWORD]}"
 
-  COMMANDS='\
-    help install uninstall use run exec \
-    alias unalias reinstall-packages \
-    current list ls list-remote ls-remote \
-    clear-cache deactivate unload \
+  COMMANDS='
```

Comment:
If you print it out, you'll see:
`\ help install uninstall use run exec \ alias unalias reinstall-packages \ current list ls list-remote ls-remote \ clear-cache d
eactivate unload \ version which`